lib/rust/ensogl/core/src/animation/frp/animation.rs

```diff
@@ -29,23 +29,28 @@ pub mod hysteretic;
 //     }
 //     Output {
 //         value  (f32),
-//         on_end (inertia::EndStatus),
+//         on_end (),
 //     }
 // }
```

Comment:
this is an edit in a commented code. Is this on purpose? Why this code is commented in the first place?

Reply:
Removed this code. I suppose it lives here from prehistoric times.